exercises/microwave/README.md

```diff
@@ -0,0 +1,37 @@
+# Microwave
+
+Convert the buttons entered on the microwave panel to their timer equivalent.
+
+Microwave timers are smart enough to know that when you press [9, 0] for seconds, you mean '01:30' in minutes.
```

Comment:
I might express this as:

```suggestion
Microwave timers are smart enough to know that when you press _90_ for seconds, you mean '01:30', which is 90 seconds.  We want to have a "smart display" that will convert this to the lowest form of minutes and seconds, rather than leaving it as 90 seconds.
```
Because my microwave does not have brackets, nor comma, I literally would press `90`.
Some microwaves will change the display to show `1:30` while others will show `01:30`, while mine will show `00:90` and count down from there.  Yet, as is written, they are all "smart enough" because there is no difference, they are both 90 seconds.

I think the focus here is display.